src/holo-test

```diff
@@ -39,9 +39,9 @@ run_testcase() {
     # setup chroot for holo run
     rm -rf -- target/
     cp -R source/ target/
+    mkdir -p target/run
+    mkdir -p target/tmp
     mkdir -p target/usr/share/holo/files
-    mkdir -p target/usr/share/holo/run-scripts
-    mkdir -p target/usr/share/holo/users-groups
```

Comment:
Removing that will break the testcases for `holo-run-scripts` and `holo-users-groups`.